assets/js/payment-methods-demo/stripe-utils/index.js

```diff
@@ -151,11 +153,106 @@ const canDoApplePay = ( paymentRequest ) => {
 	} );
 };
 
+const isNonFriendlyError = ( type ) =>
+	[
+		errorTypes.INVALID_REQUEST,
+		errorTypes.API_CONNECTION,
+		errorTypes.API_ERROR,
+		errorTypes.AUTHENTICATION_ERROR,
+		errorTypes.RATE_LIMIT_ERROR,
+	].includes( type );
+
+const getErrorMessageForCode = ( code ) => {
```

Comment:
These codes and messages, as well as the CC form fields, could definitely be reusable across gateways and imported into Stripe.

Reply:
Generally, agreed, however some code strings are gateway specific (in this case these codes are). Once I get everything functional, I do plan on going through and reviewing what components/utilities etc should be exposed by checkout for gateways to use as opposed to something the gateway provides.